feature-libs/checkout/base/components/checkout-review-submit/checkout-review-submit.component.html

```diff
@@ -1,81 +1,4 @@
 <div class="cx-review">
-  <!-- TITLE -->
```

Comment:
Content in the .ts file for this component has been left unchanged because b2b checkout extends using the base checkout component